### PR TITLE
UX: Remove unnecessary plugin outlet <span> wrapper

### PR DESCRIPTION
**Pull request by carsick**
There's already a `connectorTagName` specifying `div`, is there a need for these `span` wrappers which cannot be configured via component?

### CODE REVIEW

**Review by davidtaylorhq — APPROVED** [review]
LGTM, but we should review some popular themes before merging to make sure they aren't  depending on the `span` wrappers

**Comment by SamSaffron** [comment]
Conflicting needs a rebase

**Pull request closed without merging**
